src/dune_rpc_impl/import.ml

```diff
@@ -1 +1,8 @@
 module Dune_rpc = Dune_rpc_private
+
+let progress_of_build_event :
```

Comment:
Why put this in `Import`? Can't it live in the same module where it is used.

Reply:
It's needed by both `server.ml` and `long_poll.ml` and it didn't feel like it obviously belongs to either of those two.